cocos/particle/renderer/particle-system-renderer-data.ts

```diff
@@ -27,11 +34,13 @@ export default class ParticleSystemRenderer {
     }
 
     /**
-     * @zh 在粒子生成方式为 StrecthedBillboard 时,对粒子在运动方向上按速度大小进行拉伸。
+     * @en When the particle generation mode is StrecthedBillboard, in the direction of movement of the particles is stretched by velocity magnitude
```

Comment:
When the particle's generation mode is [[RenderMode.StrecthedBillboard]], this property will stretch the particle according to the velocity in the direction of movement of the particle